src/Microsoft.Azure.NotificationHubs/NamespaceManager.cs

```diff
@@ -466,8 +504,31 @@ public Task<NotificationHubDescription> UpdateNotificationHubAsync(NotificationH
             throw new NotImplementedException();
         }
 
-        private static string AddHeaderAndFooterToXml(string content) => $"{Header}{content}{Footer}";
+        /// <summary>Checks whether a notifications hub exists.</summary>
```

Comment:
These changes look like they're exactly the same as the ones represented in #69. We should limit this to one change-per-pr, or lump them all together.

Reply:
I'll fix this conflict while merging